### DISCUSSION TITLE
Reorderable method in table version 3

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#10466] [Help]
the reorder doesn't appear in the table, although it was working fine in the past 

![image](https://github.com/filamentphp/filament/assets/56083879/aab18679-e7dc-45cf-a455-7d45d69e9c45)

and here is the model 
![Screenshot from 2023-12-24 13-33-36](https://github.com/filamentphp/filament/assets/56083879/6770c203-d7ad-455a-92bb-2e553bf575bb)

**Comment by leandrocfe** [comment]
what is the filament version?

**Reply by Hegabovic** [reply]
version 3

**Reply by leandrocfe** [reply]
3.?.?

**Reply by Hegabovic** [reply]
"filament/filament": "^3.0-stable",

**Reply by leandrocfe** [reply]
it should work. Could you provide a repo on github to reproduce this issue?

**Reply by Hegabovic** [reply]
actually it is on a private company repo

**Reply by Hegabovic** [reply]
classified work

**Reply by leandrocfe** [reply]
There is no issues with `reorderable`, it is working here. 
Did you publish views?
Did you run `composer update` ?

**Reply by Hegabovic** [reply]
i didn't publish any views and also only run composer install
